substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/BundleSupport.java

```diff
@@ -119,11 +143,33 @@ String optionName() {
         }
     }
 
+    enum ExtendedBundleOptions {
+        dry_run,
+        container,
+        dockerfile;
+
+        static ExtendedBundleOptions get(String name) {
+            return ExtendedBundleOptions.valueOf(name.replace('-', '_'));
+        }
+
+        @Override
+        public String toString() {
+            return super.toString().replace('_', '-');
+        }
+    }
+
     static BundleSupport create(NativeImage nativeImage, String bundleArg, NativeImage.ArgumentQueue args) {
         try {
             String variant = bundleArg.substring(BUNDLE_OPTION.length() + 1);
             String bundleFilename = null;
-            String[] variantParts = SubstrateUtil.split(variant, "=", 2);
+            String extendedOptions = null;
+            String[] variantParts = SubstrateUtil.split(variant, ",", 2);
```

Comment:
Instead of splitting in first and rest (split with limit 2) split without limit and do not make the rest special

Reply:
Addressed by commit https://github.com/oracle/graal/pull/6496/commits/4b3d0c2c71cd45fd3dac28d7edf9b0c459d833a1